Lib/python/pybuffer.i

```diff
@@ -12,18 +12,43 @@
  *      }
  */
 
+/* Note that in Py_LIMITED_API case we have no choice, but to use deprecated
+ * functions, as they provides the only way to access buffer data with limited
+ * API, which doesn't include Py_buffer definition. We also disable the
+ * warnings about doing this because they're not useful in our case.
+ */
+
 %define %pybuffer_mutable_binary(TYPEMAP, SIZE)
 %typemap(in) (TYPEMAP, SIZE) {
   int res; Py_ssize_t size = 0; void *buf = 0;
+%#ifdef Py_LIMITED_API
```

Comment:
All the other files seem to have the `!Py_LIMITED_API` case first in conditionals, but in this file it's the other way around...

Reply:
I've reversed the check direction.